ZWaveLib/CommandClasses/MultiInstance.cs

```diff
@@ -160,6 +165,27 @@ public static ZWaveMessage SensorMultiLevelGet(ZWaveNode node, byte instance)
             });
         }
 
+        private NodeEvent HandleMultiChannelCapabilityReportV2(ZWaveNode node, byte[] message) {
+			if (message.Length < 5)
```

Comment:
Please, fix indentation.

Reply:
Sorry some tabs got included instead of spaces.